Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -84,6 +89,19 @@ YUI.add('ez-actionbarview-tests', function (Y) {
             );
         },
 
+        _isActionDisabled: function (actionId) {
```

Comment:
I meant in the actionbar, but it sounds good in the model. We need just need @dpobel to confirm it is not a problem layerwise.